core/src/main/java/edu/wpi/grip/core/operations/composite/PublishVideoOperation.java

```diff
@@ -80,7 +80,7 @@
 
           final int fps = socketInputStream.readInt();
           final int compression = socketInputStream.readInt();
-          final int size = socketInputStream.readInt();
+          // final int size = socketInputStream.readInt(); // Unused
```

Comment:
Consider removing? That being said, not reading the int might fail to progress the input stream to read the next bit of data. Are you sure this isn't breaking anything?

Reply:
Ah, good point. I'll have to check that